.gitignore

```diff
@@ -13,4 +13,7 @@ npm-debug.log
 builderror.log
 
 bson.sublime-project
-bson.sublime-workspace
\ No newline at end of file
+bson.sublime-workspace
+
+dist/*
```

Comment:
Do we want to permanently remove the `dist` directory and only have the files in the published version?

Reply:
I think the bundle is the one good file to keep here